server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -225,6 +228,10 @@ public RecordVersion highestSupportedRecordVersion() {
         }
     }
 
+    public boolean brokerRegistrationChangeRecordSupported() {
```

Comment:
This type tends to prefix this check with "is". For example, "isBrokerRegistrationChangeRecordSupported".